packages/vuetify/src/mixins/menuable.ts

```diff
@@ -138,11 +164,11 @@ export default Vue.extend({
 
       return top
     },
-    hasActivator () {
+    hasActivator (): string | boolean | object {
```

Comment:
`has` suggest a boolean, if it doesn't return boolean then it should

Reply:
trying to set this to boolean gives `Type 'string | boolean | object' is not assignable to type 'boolean'.
  Type 'string' is not assignable to type 'boolean'.` seems like activator can be type of string or object. not sure if thats right

Reply:
`return ... || !!this.activator || !!this.inputActivator`?